app/controllers/application_controller.rb

```diff
@@ -17,6 +17,7 @@
 # with BigBlueButton; if not, see <http://www.gnu.org/licenses/>.
 
 class ApplicationController < ActionController::Base
+  include ApplicationHelper
```

Comment:
Helpers should only contain code that is used in the frontend

Reply:
You got reason.
My bad, I'll make it an `ApplicationController` method.